dp/cloud/helm/dp/Chart.yaml

```diff
@@ -0,0 +1,67 @@
+---
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+appVersion: "0.1.0"
+version: 0.1.0
+description: A Helm chart for magma orchestrator's domain-proxy module.
+name: domain-proxy
+engine: gotpl
+sources:
+  - https://github.com/magma/magma
+keywords:
+  - magma
+  - or8cr
+  - dp
+type: application
+
+long_description: |
+  This Chart will deploy the following:
+
+  - 1 x Configuration controller.
+  - 1 x protocol controller with 80/443 TCP port exposed with either ingress/contour ingress resource.
+  - 1 x radio controller with GRPC 50053/TCP port.
+  - All using Kubernetes Deployment
+  - 1 x job for Postgres database migration.
+
+  ## Installation.
+
+  ### Preprequsites.
+
+  Domain proxy requires an ingress controller to be present on targer kubernetes cluster.
```

Comment:
Hi @hcgatewood ,
nginx was picked because of it's widespread adoption, good documentation, and big community. Contour on the other hand underneath uses envoy proxy, which is also used by ambassador, which we can later move on. DP expects ingress controller to be present, and I think that whether it should be enabled for whole magma project is a different, much bigger discussion. That being said, adding it on fresh kubernetes cluster is a matter of update existing terraform code, which can be done in separate PR. Of course I can prepare also config for current nginx pod solution.